impeller/entity/contents/filters/gaussian_blur_filter_contents.h

```diff
@@ -35,9 +36,12 @@ class GaussianBlurFilterContents final : public FilterContents {
   static std::string_view kNoMipsError;
   static const int32_t kBlurFilterRequiredMipCount;
 
-  explicit GaussianBlurFilterContents(Scalar sigma_x,
-                                      Scalar sigma_y,
-                                      Entity::TileMode tile_mode);
+  explicit GaussianBlurFilterContents(
+      Scalar sigma_x,
+      Scalar sigma_y,
+      Entity::TileMode tile_mode,
+      BlurStyle blur_style,
+      const std::shared_ptr<Geometry>& geometry);
```

Comment:
uber-nit: I might call this something like "mask_geometry", to clarify that it is the geometry for the mask and not the destination geometry/rect of the filter.